backend/src/main/resources/db/migration/V2__registration_entities.sql

```diff
@@ -0,0 +1,69 @@
+--  modify old user entity
+ALTER TABLE `user`
```

Comment:
My suggestion here is to use multiple script for every alter, every create, every insert...

Reply:
Is it enough to break the whole script into:
- modify user entity
- create role
- create company
- add foreign keys
- init role and company

In other words, break it up where I've left comments.

Or should I break it down further?

Reply:
I would say script per alter, script per create, scrip per insert (you can use one insert for multiple values in same table)

Reply:
Ok, I'll make one script per statement.

Reply:
In that case, scripts will have understandable names and we will know which script does what just by reading the name